docs/src/paradox/03-apis/api-v2/query-language.md

```diff
@@ -138,6 +138,37 @@ main resources and those for B as dependent resources. The main resource
 must be represented by a variable, marked with `knora-api:isMainResource`,
 as explained under @ref:[CONSTRUCT Clause](#construct-clause).
 
+## Virtual incoming Links
```

Comment:
Does API v2 do this in all resource responses, not just in Gravsearch responses? If so, should this section be in `reading-and-searching-resources.md`?

Reply:
done in 
https://github.com/dhlab-basel/Knora/pull/1070/commits/33cc8f8e519eac5c6d05df244890b34224512963